src/ecalc/libraries/libecalc/common/libecalc/core/models/compressor/train/variable_speed_compressor_train_common_shaft.py

```diff
@@ -533,7 +534,7 @@ def _calculate_train_result_given_rate_ps_speed(
             )
 
         choked_inlet_pressure = find_root(
-            lower_bound=LOWEST_POSSIBLE_CHOKE_PRESSURE_BARA,
+            lower_bound=UnitConstants.STANDARD_PRESSURE_BARA + self.stages[0].pressure_drop_ahead_of_stage,
```

Comment:
I guess we can change to `UnitConstants.STANDARD_PRESSURE_BARA` the other place where `LOWEST_POSSIBLE_CHOKE_PRESSURE_BARA` is used as well, and remove `LOWEST_POSSIBLE_CHOKE_PRESSURE_BARA`.